panels/dev-service/ha-panel-dev-service.html

```diff
@@ -49,7 +49,7 @@
       }
 
       .attributes tr:nth-child(even) {
-        background-color: #eee;
+        background-color: var(--paper-listbox-background-color,#eee)
```

Comment:
Actually we should use a different variable here, something like: 
--table-row-alternative-background-color

And maybe add:

.attributes tr:nth-child(odd) {
         background-color: var(--table-row-background-color)
}

 We should also change that in dev-state.